index.js

```diff
@@ -110,10 +112,12 @@ class Collection {
     return this.collection;
   }
 
-  static getCollectionBySlug(client, slug, params) {
+  static getCollectionBySlug(client, slug, params, options = {}) {
+    const {loadNestedCollections = true, depth = DEFAULT_DEPTH} = options;
```

Comment:
Two things
1) Just merge params and options. (But not mandatory)
2) Why two parameters? Isn't `loadNestedCollections` the same as `depth > 0`